test/e2e/common/cleanup.sh

```diff
@@ -16,12 +16,53 @@
 set -x # echo so that users can understand what is happening
 set -e # exit on error
 
+
+env="kind"
+
+if [ "$1" == "--env" ]; then
+    env="$2"
+    shift
```

Comment:
Consuming two command line arguments would normally be done with _two_ `shift` commands.
This is not a fatal problem at the moment, because nothing later looks at the command line. But this is a fragile comfort.

Reply:
Fixed